dist/SectionsContainer.js

```diff
@@ -365,7 +371,7 @@ var SectionsContainer = function (_React$Component) {
                     transform: _this5.state.activeSection === index ? 'scale(1.3)' : 'none'
                 };
 
-                return React.createElement('a', { href: '#' + link, key: index, className: (_this5.state.activeSection===index) ? 'Navigation-Anchor '+_this5.props.activeClass : 'Navigation-Anchor',
+                return React.createElement('a', { href: '#' + link, key: index, className: _this5.props.navigationAnchorClass || 'Navigation-Anchor',
```

Comment:
Not sure what is happening here. I only expected changes at lines 59 and 158-162, but this code was also changed during the build process.